jsonrpc/txpool_endpoint.go

```diff
@@ -0,0 +1,153 @@
+package jsonrpc
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/0xPolygon/minimal/types"
+)
+
+// Txpool is the txpool jsonrpc endpoint
+type Txpool struct {
+  d *Dispatcher
+}
+
+type ContentResponse struct {
+  Pending   map[types.Address]map[uint64]*gethTransaction   `json:"pending"`
+  Queued    map[types.Address]map[uint64]*gethTransaction   `json:"queued"`
+}
+
+type InspectResponse struct {
+  Pending   map[string]map[string]string  `json:"pending"`
+  Queued    map[string]map[string]string  `json:"queued"`
+}
+
+type StatusResponse struct {
+  Pending   uint64  `json:"pending"`
+  Queued    uint64  `json:"queued"`
+}
+
+type gethTransaction struct {
```

Comment:
I'm not sure about the naming on this. `gethTransaction` implies that the transaction is conforming to specific geth services, when in fact it is just emulating the same functionality.